app/javascript/packs/media.js

```diff
@@ -15,4 +15,25 @@ $(document).ready(function() {
   CssInjection.appendToHead();
   CommonViewerBehavior.initializeViewer();
   EmbedThis.init();
+
+  const tabs = document.querySelectorAll('[role="tab"]')
+
+  // Add a click event handler to each tab
+  tabs.forEach((tab) => {
+    tab.addEventListener("click", changeTabs);
+  });
 });
+
+function changeTabs(e) {
+  const target = e.target;
+  const parentButton = target.parentNode;
```

Comment:
This seems a bit fragile as it's tightly bound to the structure of the dom.  Can you use `.closest('[role="tabitem"]')` here and same for `tablist` below?

Reply:
Sure - I think this makes sense for the tablist select, I'm curius about this for parent... the issue is that the click event happens on the svg object, not the button, so I need to grab the parent button of the svg item... would `closest` be sufficient? And is `tabitem` correct vs `tab`?

I'll work through it and see what happens.

Reply:
Figured it out, I think it does need to be `tab` :-)